scripts/tools/InitializeQueue.php

```diff
@@ -159,8 +172,8 @@ private function checkParams(array $params)
 
             switch ($option) {
                 case '--broker':
-                    if (!in_array($value, [self::BROKER_MEMORY, self::BROKER_RDS, self::BROKER_SQS])) {
-                        throw new \InvalidArgumentException('Broker "' . $value . '" is not a valid broker option. Valid options: ' . implode(', ', [self::BROKER_MEMORY, self::BROKER_RDS, self::BROKER_SQS]));
+                    if (!in_array($value, self::AVAILABLE_BROKERS)) {
+                        throw new \InvalidArgumentException('Broker "' . $value . '" is not a valid broker option. Valid options: ' . implode(', ', self::AVAILABLE_BROKERS));
                     }
 
                     $this->wantedBroker = $value;
```

Comment:
- [ ] Please validate that the `newSql` broker requires a persistence too in the method `checkParams`.